app/controllers/anime_controller.rb

```diff
@@ -185,13 +203,12 @@ def filter_new
           @years = @filter_years
         end
 
-        if params[:sort]
-          @sort = params[:sort]
-        else
-          @sort = "all"
-        end
+        @sort = params[:sort] ||= 'all'
 
-        @anime = Anime.accessible_by(current_ability).references(:genres).page(params[:page]).per(36)
+        @anime = Anime.accessible_by(current_ability)
```

Comment:
We can probably remove the `accessible_by` here, since we're no longer using Cancan